react/features/calendar-sync/components/MeetingList.native.js

```diff
@@ -237,8 +282,11 @@ class MeetingList extends Component<Props> {
  * }}
```

Comment:
These comments are so problematic, yeah? I wonder if there's a tool or IDE that could help cuz I miss updating these often. This should be updated for _calendarAccessStatus.